modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -866,6 +879,13 @@ def test_drop_duplicates_after_sort():
     df_equals(modin_result, pandas_result)
 
 
+def test_drop_duplicates_with_repeated_index_values():
```

Comment:
Could you please add a link to the GitHub issue?

Reply:
I pushed a new commit with the link as a comment. Is that what you're referring to or is there some better way to link the issue?

Reply:
That's what I was thinking of. Thank you!